gamebench_api_client/api/requests_retriever/builder/attributes/attributes_creator.py

```diff
@@ -36,8 +39,9 @@ def get_data(self):
                 'username': self.request_parameters['username'],
                 'password': self.request_parameters['password']
             }
-        else:
-            return self.request_parameters["data"]
+        elif 'data' not in self.request_parameters:
+            self.request_parameters['data'] = ''
```

Comment:
What do you think about either added an else block that does nothing, or an explanatory comment for why there isn't an else block?

Reply:
Disregard, instead of a check on 'data', use change:
return self.request_parameters['data']
to:
return self.request_parameters.get('data', '')

Reply:
Updated both the get_data and get_params methods to use the .get()